### ISSUE TITLE
non-finite param for setTargetAtTime

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/friction#154]
This error is happening on bayes CT with ?fuzzBoard. But I can also reproduce by trying to drag a book with the keyboard.

```js
Uncaught TypeError: Failed to execute 'setTargetAtTime' on 'AudioParam': The provided float value is non-finite.
    at BookRubSoundGenerator.setOutputLevel (SoundGenerator.js?bust=1544025792025:189)
    at BookRubSoundGenerator.Property.multilink (BookRubSoundGenerator.js?bust=1544025792025:88)
    at Array.listener (Multilink.js?bust=1544025792025:43)
    at Emitter.emit3 (Emitter.js?bust=1544025792025:265)
    at NumberProperty._notifyListeners (Property.js?bust=1544025792025:287)
    at NumberProperty.setValueAndNotifyListeners (Property.js?bust=1544025792025:265)
    at NumberProperty.set (Property.js?bust=1544025792025:198)
    at NumberProperty.set (NumberProperty.js?bust=1544025792025:84)
    at Array.BookRubSoundGenerator.topBookPositionProperty.lazyLink (BookRubSoundGenerator.js?bust=1544025792025:57)
    at Emitter.emit3 (Emitter.js?bust=1544025792025:265)
```
Assigning to @jbphet and @zepumph since this is related to both audio and alternative input, but let me know if I can help or if this seems related to recent changes for a11y input in scenery.

**Comment by zepumph** [comment]
@jbphet let me know if you need help.

**Comment by jbphet** [comment]
I fixed this before noticing that CT had been impacted.  The commit can be found here: https://github.com/phetsims/friction/commit/95dcb993b0ebeb6a75404460beffb9436a7a3081.  Closing.
